package.json

```diff
@@ -6,20 +6,38 @@
   "description": "Vue components, Vue directives, Design Token and Scss files to build interfaces for demosPlan.",
   "main": "./dist/demosplan-ui.umd.js",
   "dependencies": {
+    "@babel/core": "^7.20.2",
```

Comment:
there is a devDependency-Section again. f You update the branch with main, you will see it.